pkg/tfgen/docs.go

```diff
@@ -402,7 +402,12 @@ func parseTFMarkdown(g *generator, info tfbridge.ResourceOrDataSourceInfo, kind
 		}
 	}
 
-	return cleanupDoc(g, info, ret), nil
+	doc, err := cleanupDoc(g, info, ret)
+	if err != nil {
```

Comment:
If we’re going to do this unconditionally on an error, why not just diag warn the error itself?